moulinette/interfaces/__init__.py

```diff
@@ -156,7 +154,8 @@ def parse_args(self, args, **kwargs):
 
     # Arguments helpers
 
-    def prepare_action_namespace(self, tid, namespace=None):
+    @staticmethod
```

Comment:
Why have you made this method static? It's because "self" isn't used inside?

Reply:
Uuh yes, is it bad?

Reply:
?